acid-view/src/CmdParse.hs

```diff
@@ -0,0 +1,61 @@
+module CmdParse where 
+
+import Universum
+import Options.Applicative
+
+data TypeOfOutput = Wallets | Accounts | Addresses | AllData
+
+data CmdOptions = CmdOptions
+    {  path       :: FilePath
+    ,  position   :: Int 
+    ,  output     :: TypeOfOutput
+    ,  fileToSave :: Maybe FilePath
+    }
+
+
+cmdArgToTypeOfOutput :: String -> TypeOfOutput 
+cmdArgToTypeOfOutput output = case output of 
+    "wal"  -> Wallets 
+    "acc"  -> Accounts
+    "addr" -> Addresses
+    "all"  -> AllData
+    _      -> AllData 
+
+opts :: ParserInfo CmdOptions
+opts = info (fileOptions <**> helper)
+      ( fullDesc
+     <> progDesc "Show acid state db contents"
+     <> header "Show acid state db contents" )
+
+parsedCMDOptions :: IO CmdOptions
+parsedCMDOptions = execParser opts
+
+fileOptions :: Parser CmdOptions 
+fileOptions = CmdOptions 
+  <$> strOption 
+        ( long "dbpath"
+       <> metavar "DBPath"
+       <> showDefault
+       <> value "~/.local/share/ariadne/.wallet-db/" 
+       <> help "Path with acid db files")
+  <*> option auto 
+        ( long "position"
+       <> short 'p'
+       <> showDefault 
+       <> value 0 
+       <> help "number of state"
+        )  
+  <*> (cmdArgToTypeOfOutput <$> strOption 
+       ( long "show"
+      <> short 's'
+      <> showDefault
+      <> value "all" 
+      <> help "What do you want to see ? wal -> only wallets, acc -> only accounts, addr -> only addresses, all -> all of this"))

```

Comment:
Ok, changed to `"What do you want to see? ("wal"(Wallets) | "acc"(Accounts) | "addr"(Addresses) | "all"(AllData))"`, because i want to save short writing.